check_http_expect

```diff
@@ -153,11 +154,13 @@ while [ $# -gt 0  ]; do
 		# Final target url
 		--url)
 			url="$1"
+			shift
```

Comment:
`shift` is already happening here at the top of the loop: https://github.com/cytopia/check_http_expect/pull/2/files#diff-dd5699a63f65e4bc94d8545accf83734R151

The current implementation will break parameter parsing.

Try out:
```
$ ./check_http_expect --url http://google.de

Error, You must specify what you are looking for.
Usage:  check_http_expect --url <url> --find <string> [--insecure] [--huser <user>] [--hpass <pass>] [--lurl <url>] [--cookie <data> [--cookie <data>]][--ldata <data> [--ldata <data>]]
OR      check_http_expect --check
OR      check_http_expect --help
OR      check_http_expect --version
```

Reply:
I removed the shift at the bottom to compensate for this, so we can have a flag that doesn't need a parameter. Are you sure the parsing is broken? That error message to me implies you forgot to set --find which is a required flag as well.

Reply:
Looks like you're absolutely right. That's something I didn't catch in the review and also didn't read the error message properly. :-)